packages/web-api/src/WebClient.spec.ts

```diff
@@ -1190,7 +1190,11 @@ describe('WebClient', () => {
           file_id: 'F0123456789',
           upload_url: 'https://files.slack.com/upload/v1/abcdefghijklmnopqrstuvwxyz',
         })
-        .post('/api/files.completeUploadExternal', { files: '[{"id":"F0123456789","title":"test-txt.txt"}]' })
+        .post('/api/files.completeUploadExternal', {
+          blocks: '[{"type":"section","text":{"type":"plain_text","text":"Hello"}}]',
+          channel_id: 'C010101010',
+          files: '[{"id":"F0123456789","title":"test-txt.txt"}]',
+        })
```

Comment:
Since `blocks` is optional should we leave this unit test as it is and move this to its own test? 🤔

Reply:
@WilliamBergamin Super good call! It also gave us a chance to test `content` in a more complete unit with the https://github.com/slackapi/node-slack-sdk/pull/2261/commits/e23b0660b1160d68a52026ba9903f8c33871db0e changes 🧪 ✨